src/_gettsim/social_insurance_contributions/ges_krankenv.py

```diff
@@ -674,28 +674,28 @@ def _ges_krankenv_bruttolohn_m(
     return out
 
 
-def _ges_krankenv_beitr_reg_beschäftigt_m(
-    _ges_krankenv_bruttolohn_m: float, ges_krankenv_beitr_satz: float
+def _ges_krankenv_beitr_arbeitnehmer_reg_beschäftigt_m(
+    _ges_krankenv_bruttolohn_m: float, ges_krankenv_beitr_satz_arbeitnehmer: float
 ) -> float:
-    """Calculate health insurance contributions for regular jobs.
+    """Calculate employee's health insurance contributions for regular jobs.
 
     Parameters
     ----------
     _ges_krankenv_bruttolohn_m
         See :func:`_ges_krankenv_bruttolohn_m`.
-    ges_krankenv_beitr_satz
-        See :func:`ges_krankenv_beitr_satz`.
+    ges_krankenv_beitr_satz_arbeitnehmer
+        See :func:`ges_krankenv_beitr_satz_arbeitnehmer`.
     Returns
     -------
     Pandas Series containing monthly health insurance contributions for regularly
```

Comment:
While you are at it: can you S+R for "pandas series" in docstrings? Those are from ages ago...